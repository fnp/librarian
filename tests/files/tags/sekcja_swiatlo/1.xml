<akap>Tekst</akap>

<sekcja_swiatlo />

<akap>Tekst</akap>
