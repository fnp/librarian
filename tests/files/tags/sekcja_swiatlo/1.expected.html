<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Tekst</p>
<hr class="spacer">
<a href="#f2" class="wl-num">2</a><p class="wl paragraph" id="f2">Tekst</p>
</div>
